.cspell.json

```diff
@@ -245,5 +260,6 @@
     ".gitignore",
     "angular.json",
     "ng-package.json"
-  ]
+  ],
+  "allowCompoundWords": true
```

Comment:
To allow this to not be flagged:
https://github.com/novuhq/novu/blob/36ba8e1c6e6f7d1e1b44357f963c7916af47b048/libs/dal/src/repositories/topic/topic.repository.ts#L34

Reply:
Can we maybe just use an ignore for the particular case in the code, because for future it might catch good un intentional compound words where they could just been a `camelCase` variable for example. I think collection names are one of those exceptions for the rule, but i think better leave this is on for all other cases as collection name are quite edge cases in our repo.

Reply:
Ok, I can do that, but the dictionary can grow a lot, that's why I went for this property. Also I think it only checks strings inside variables, not the variable naming itself.